core/server/common/src/main/java/alluxio/master/journal/raft/JournalStateMachine.java

```diff
@@ -170,8 +172,8 @@ private synchronized void applyToMaster(JournalEntry entry) {
       LOG.trace("Applying entry to master {}: {} ", masterName, entry);
       master.processJournalEntry(entry);
     } catch (Throwable t) {
-      ProcessUtils.fatalError(LOG, t, "Failed to apply journal entry to master %s. Entry: %s",
-          masterName, entry);
+      JournalUtils.handleJournalReplayFailure(LOG, t,
+          "Failed to apply journal entry to master %s. Entry: %s", entry);
```

Comment:
NIT: the string format has 2 placeholders, but only 1 argument.

Reply:
Added the masterName argument, Thanks for finding this error!